src/components/tables/torrenttable.tsx

```diff
@@ -299,13 +299,24 @@ function ByteRateField(props: TableFieldProps) {
 }
 
 function PercentBarField(props: TableFieldProps) {
-    const now = props.torrent[props.fieldName] * 100;
+    let now: number = props.torrent[props.fieldName] * 100;
+    let label: string = '';
+    let status: number = props.torrent.status;
+    if ((props.torrent.error !== undefined && props.torrent.error > 0) || props.torrent.cachedError !== "") {
+        status = Status.error;
+    } else if (props.torrent.status === Status.downloading && props.torrent.pieceCount === 0) {
+        now = 100;
```

Comment:
Not sure if this is leftover from testing or you intend to always show full progress bar. I'm inclined to not show it, even if it makes the color of the progress bar rarely visible because it usually goes from 0 to downloading in a single update.

Reply:
The problem with the Magnetizing one is that I think you said the Magnetization progress is not grabbed at the full table level, only at the per-torrent level. So the Magnetization process will _never_ be shown. I felt it's better to have the bar be full all the time to at least show that this torrent is being Magnetized.

Reply:
Makes sense, showing full animated bar with no percentage text will be fine.